src/test/rustdoc-gui/anchors.goml

```diff
@@ -1,17 +1,29 @@
+// This test is to ensure that the anchors (`§`) have the expected color.
 goto: file://|DOC_PATH|/test_docs/struct.HeavilyDocumentedStruct.html
 
+// This is needed to ensure that the text color is computed.
+show-text: true
```

Comment:
This was originally missing, meaning that the colors weren't computed. I'll add a failure in `browser-ui-test` directly.